ffi/capi/src/fixed_decimal.rs

```diff
@@ -22,15 +22,36 @@ pub extern "C" fn icu4x_fixed_decimal_create(magnitude: i64) -> *mut ICU4XFixedD
     Box::into_raw(Box::new(fd))
 }
 
+#[repr(C)]
+/// This is the result returned by [`icu4x_fixed_decimal_multiply_pow10()`]
+pub struct ICU4XFixedDecimalMultiplyPow10Result {
+    /// Whether the multiplication was successful.
+    pub success: bool,
+    /// The error type if the multiplication failed.
+    pub error_code: i32,
```

Comment:
I would use a C-like enum that has an explicit discriminant instead (which includes a `Success = 0` variant) so that the C API can consume this nicely. I'd also give it a From impl from the regular one.